dastardcommander/dc.py

```diff
@@ -320,10 +322,14 @@ def updateReceived(self, topic, message):
             elif topic == "CHANNELNAMES":
                 self.channel_names[:] = []  # Careful: don't replace the variable
                 self.channel_prefixes.clear()
-                for name in d:
+                self.channel_indices.clear()  # a map from channel numbers to indices
+                for index, name in enumerate(d):
                     self.channel_names.append(name)
                     prefix = name.rstrip("1234567890")
                     self.channel_prefixes.add(prefix)
+                    if prefix == "chan":
+                        number = int(name[len(prefix):])
```

Comment:
`len(prefix)` could be hardcoded as `4` given that we know it is `=="chan"`. Is that any better?

Reply:
Not enough to bother with!